recipes/cppast/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set name = "cppast" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/foonathan/cppast/archive/refs/tags/v{{ version }}.zip
+  sha256: 06f11c7a86263f10187ce45977e78c53b597032b4d24f501cd54f41ceae53721
+  patches:
+    - 0001-do-not-vendor-external-dependencies.patch
+    # Upstreamed at https://github.com/foonathan/cppast/pull/145
+    - 0002-install-library-binary.patch
+    - 0003-simplify-cmake.patch
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
```

```suggestion
build:
  number: 0
  skip: true  # [win and vc<14]
  run_exports:
    - {{ pin_subpackage('cppast', max_pin='x.x.x') }}
  ignore_run_exports_from:
    - zlib  # [win]
```
You are installing headers, so you also need a run_export based on *ABI* compatability guarantees of this package. It seems to be in early development, so I recommend pinning to patch version and re-evaluating at version 1.0.

zlib is needed for headers (as noted), but the link inspector says it is not linking against the shared library, so lets ignore run_exports from zlib.

---

Sounds good.

---

Ok. My point was that a header-only library would not need the run exports (at least not if things are namespaced properly in the headers.)